docs/bootedge.md

```diff
@@ -0,0 +1,149 @@
+# Visual Inspection AI Edge Solution
```

Comment:
There's no need to repeat this heading across several docs. Can you please remove it from other pages as well? thanks!